public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -262,14 +262,19 @@ public ConceptAnalysis apply(ConceptAnalysis ca) {
             standardConceptFilter = StandardConceptFilter.STANDARD_OR_CODE_ID_MATCH;
         }
 
+        List<String> domainIds = null;
+        if (searchConceptsRequest.getDomain() != null) {
+            domainIds = DOMAIN_MAP.get(searchConceptsRequest.getDomain()).asList();
+        }
+
         if(searchConceptsRequest.getQuery() == null || searchConceptsRequest.getQuery().isEmpty()){
             List<Concept> concepts;
             if(standardConceptFilter == StandardConceptFilter.STANDARD_CONCEPTS){
```

Comment:
Do we have tests for the various permutations of the searchRequest and filters?    We need tests for those. This one will probably fail if standard concept filter = STANDARD_OR_CODE_ID_MATCH  ... We will get all concepts I bet including non standard .